spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,16 @@
+{
+  "srcDir": "public/assets",
+  "srcFiles": [
+    "govuk-admin-template-*.js",
+    "application-*.js"
+  ],
+  "cssFiles": [
+    "application-*.css"
+  ],
+  "specDir": "spec/javascripts",
+  "specFiles": [
+    "**/*[sS]pec.js"
+  ],
+  "helpers": [],
+  "browser": "headlessChrome"
+}
```

Comment:
Looks like we're missing a new line character here - my pro tip would be to configure your text editor to always leave a new line on saving.